containers/Connector/Connector.tsx

```diff
@@ -49,13 +67,11 @@ const useConnector = () => {
 				process.env.NEXT_PUBLIC_INFURA_PROJECT_ID
 			);
 
-			synthetix.setContractSettings({
-				networkId,
-				provider,
-			});
+			const snxjs = synthetix({ provider, networkId });
```

Comment:
I thought we were putting this in context?

Reply:
oh, it is just initialized here for context. got it

Reply:
`Connector` is basically a context, AFAIK